modcc/expression.cpp

```diff
@@ -1138,6 +1138,8 @@ expression_ptr binary_expression(Location loc,
         case tok::lte      :
         case tok::gt       :
         case tok::gte      :
+        case tok::and_t    :
```

Comment:
What was the logic behind calling them `_t` (other than `and` and `or` being reserved keywords)?
How about `tok::land` and `tok::lor` for "logical and" and "logical or" respectively?

Reply:
That works.